x-pack/test/functional/apps/ml/anomaly_detection/aggregated_scripted_job.ts

```diff
@@ -0,0 +1,476 @@
+          await ml.testExecution.logTestStep('should display the chart');
+          await ml.singleMetricViewer.assertChartExist();
+
+          await ml.testExecution.logTestStep('should navigate to anomaly explorer');
+          await ml.navigation.navigateToAnomalyExplorerViaSingleMetricViewer();
+
+          await ml.testExecution.logTestStep('pre-fills the job selection');
+          await ml.jobSelection.assertJobSelection([testData.jobConfig.job_id]);
+
+          await ml.testExecution.logTestStep('displays the swimlanes');
+          await ml.anomalyExplorer.assertOverallSwimlaneExists();
+          await ml.anomalyExplorer.assertSwimlaneViewByExists();
+        });
+      });
+    }
+
+    for (const testData of unsupportedTestSuites) {
+      describe(testData.suiteTitle, function () {
+        before(async () => {
+          await ml.api.createAndRunAnomalyDetectionLookbackJob(
+            testData.jobConfig,
+            testData.datafeedConfig
+          );
+        });
+
+        it('opens a job from job list link', async () => {
+          await ml.testExecution.logTestStep('navigate to job list');
+          await ml.navigation.navigateToMl();
+          await ml.navigation.navigateToJobManagement();
+
+          await ml.testExecution.logTestStep(
+            'check that the single metric viewer button is disabled'
+          );
+          await ml.jobTable.waitForJobsToLoad();
+          await ml.jobTable.filterWithSearchString(testData.jobConfig.job_id, 1);
+
+          await ml.jobTable.assertJobActionSingleMetricViewerButtonEnabled(
+            testData.jobConfig.job_id,
+            false
+          );
+
+          await ml.testExecution.logTestStep('open job in anomaly explorer');
+          await ml.jobTable.clickOpenJobInAnomalyExplorerButton(testData.jobConfig.job_id);
+          await ml.commonUI.waitForMlLoadingIndicatorToDisappear();
+        });
+
+        it('displays job results', async () => {
+          await ml.testExecution.logTestStep('pre-fills the job selection');
+          await ml.jobSelection.assertJobSelection([testData.jobConfig.job_id]);
+
+          await ml.testExecution.logTestStep('displays the swimlanes');
+          await ml.anomalyExplorer.assertOverallSwimlaneExists();
+          await ml.anomalyExplorer.assertSwimlaneViewByExists();
+
+          // TODO: click on swimlane cells to trigger warning callouts
+          // when we figure out a way to click inside canvas renderings
+
+          await ml.testExecution.logTestStep('should navigate to single metric viewer');
+          await ml.navigation.navigateToSingleMetricViewerViaAnomalyExplorer();
+
+          await ml.testExecution.logTestStep(
+            'should show warning message and redirect single metric viewer to another job'
+          );
+          await ml.singleMetricViewer.assertToastMessageExists(
+            'mlTimeSeriesExplorerDisabledJobReasonWarningToast'
```

Comment:
I'd like to move the test subject value out of the test file into a service method, maybe something like `assertDisabledJobReasonWarningToastExist`.

Reply:
Updated here c95a44a